Gems/Atom/RHI/DX12/Code/Source/RHI/Device.cpp

```diff
@@ -61,6 +100,11 @@ namespace AZ
                 ReleaseQueue::Descriptor releaseQueueDescriptor;
                 releaseQueueDescriptor.m_collectLatency = m_descriptor.m_frameCountMax - 1;
                 m_releaseQueue.Init(releaseQueueDescriptor);
+
+                Dx12maReleaseQueue::Descriptor dx12maReleaseQueueDescriptor;
+                dx12maReleaseQueueDescriptor.m_collectLatency = m_descriptor.m_frameCountMax - 1;
```

Comment:
I dont know why we use m_descriptor.m_frameCountMax - 1 instead of m_descriptor.m_frameCountMax for releasequeues in dx12 and vk rhi. I think we should switch to latter.